scripts/Follower.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python2
+import rospy
+import math
+from geometry_msgs.msg import PointStamped
+from ackermann_msgs.msg import AckermannDrive
+from std_msgs.msg import String
+from state_controller.msg import TwistLabeled
+from tf import TransformListener
+
+class Follower():
+    def __init__(self):
+        rospy.init_node('Follower', anonymous=True)
+        self.subPoint = rospy.Subscriber('/point2follow', PointStamped, self.callback)
+        self.pubDrive = rospy.Publisher('/state_controller/cmd_behavior_twist',TwistLabeled, queue_size=10)
+        self.point = None
+        self.rate = rospy.Rate(1)
+        self.tf = TransformListener()
+
+    def callback(self,data):
+        #self.point = self.tf.transformPoint('/hood',data).point
+        self.point = data.point
+        rospy.loginfo(self.point)
+
+    def follow(self):
+        drive_msg = TwistLabeled()
+        drive_msg.label = String()
+        drive_msg.label.data = "2D Follower"
+        distance = math.sqrt(self.point.x**2+self.point.y**2)
+        if (distance > 1): #If obstacle is far away, go fast
+            speed = 0.25 * (distance - 1) 
+        else: # if obstacle is really close, stop moving
+            speed = 0
+        if(speed > 1):
+            speed = 1
+        drive_msg.twist.linear.x = speed
+        angle = math.atan2(self.point.y,self.point.x) # Finds the angle at which the tractor should turn
+        angle = math.degrees(angle) # set the angle to the ackermann message
+        if(angle > 45):
+            angle = 45
+        elif(angle < -45):
+            angle = -45
+        elif(distance < 1): 
```

Comment:
Not sure that the tractor should attempt to steer back to straight if the object is close . . . Probably, when an object is close, we shouldn't attempt to move the tires quickly.